### PR TITLE
Support -f and -p options for plot creation via daemon API

### PR DESCRIPTION
**Pull request by ChiaMineJP**
It should be able to post `-f` or `-p` options for `chia plots create` command through daemon API.
I guess why those options are missing is that because creating plots from GUI has not yet supported farmer key nor pool contract address instead of wallet fingerprint.
But pool mining is upcoming and we should prepare to plot via RPC/Websocket.
So adding `-f` and `-p` handling in `daemon` service is required for now.

```python
    def _build_plotting_command_args(self, request: Any, ignoreCount: bool) -> List[str]:
        service_name = request["service"]

        k = request["k"]
        n = 1 if ignoreCount else request["n"]
        t = request["t"]
        t2 = request["t2"]
        d = request["d"]
        b = request["b"]
        u = request["u"]
        r = request["r"]
        f = request["f"] # <-- This should be implemented
        p = request["p"] # <-- This should be implemented
        a = request.get("a")
        e = request["e"]
        x = request["x"]
        override_k = request["overrideK"]

        command_args: List[str] = []
        command_args += service_name.split(" ")
        command_args.append(f"-k{k}")
        command_args.append(f"-n{n}")
        command_args.append(f"-t{t}")
        command_args.append(f"-2{t2}")
        command_args.append(f"-d{d}")
        command_args.append(f"-b{b}")
        command_args.append(f"-u{u}")
        command_args.append(f"-r{r}")

        if a is not None:
            command_args.append(f"-a{a}")

        if f is not None:
            command_args.append(f"-f{f}")

        if p is not None:
            command_args.append(f"-p{p}")

        if e is True:
            command_args.append("-e")

        if x is True:
            command_args.append("-x")

        if override_k is True:
            command_args.append("--override-k")

        self.log.debug(f"command_args are {command_args}")

        return command_args
```

### CODE REVIEW

**Review by wjblanke — DISMISSED** [review]
aok and thanks!

File: chia/daemon/server.py [hunk 1]
```diff
@@ -396,6 +396,8 @@ def _build_plotting_command_args(self, request: Any, ignoreCount: bool) -> List[
         b = request["b"]
         u = request["u"]
         r = request["r"]
+        f = request["f"]
+        p = request["p"]
```
Comment (wjblanke) [inline chia/daemon/server.py:400]: I think these need a .get here actually

Comment (ChiaMineJP) [reply chia/daemon/server.py:400]: Looking how '-a' option is treated, yes.

**Comment by ChiaMineJP** [comment]
Sorry, somehow I dismissed stale review from this thread.
Note: I've added "-c" option handling for upcoming pool protocol.

**Review by wjblanke — DISMISSED** [review]
aok

**Comment by ChiaMineJP** [comment]
Fixed an ultra minor lint error.
Sorry for botheing you in a short period of time.

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**